service/hook/endpoint.go

```diff
@@ -171,10 +172,23 @@ func HTTPHandler(w http.ResponseWriter, r *http.Request) {
 	respondWith := hookCommon.TransformResponseInputModel{
 		Errors:                  []string{},
 		SuccessTriggerResponses: []bitriseapi.TriggerAPIResponseModel{},
+		SkippedTriggerResponses: []bitriseapi.SkipAPIResponseModel{},
 		FailedTriggerResponses:  []bitriseapi.TriggerAPIResponseModel{},
 	}
 	metrics.Trace("Hook: Trigger Builds", func() {
 		for _, aBuildTriggerParam := range hookTransformResult.TriggerAPIParams {
+			commitMessage := aBuildTriggerParam.BuildParams.CommitMessage
+
+			if strings.Contains(commitMessage, "[skip ci]") || strings.Contains(commitMessage, "[ci skip]") {
```

Comment:
move this into a function - and write tests for it!

write both "yes, should skip" and "no, this should not be skipped" tests.

Example commit messages to test against:

* `this should be [ci skip]ped`
* `this will not be [CI SKIP]ped`
* `this won't be skipped either: [ ci  skip ]`
* `[skip ci] will skip`
* ...

We use unit tests to document the code, so you should try to add tests to demonstrate both what will be skipped and what won't be skipped, including possible edge cases (e.g. case sensitivity).